nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/SplitText.java

```diff
@@ -326,6 +329,27 @@ public void process(InputStream in) throws IOException {
         return splitFlowFiles;
     }
 
+    /**
+     * Will merge the two files. This operation has different semantics then
+     * sessioin.merge. Also, it always expects 2 and no more then 2 flow files,
```

Comment:
Typo `sessioin`. If we have cycle to update this PR, I'd prefer having this method suggested as below.